datalad/support/exceptions.py

```diff
@@ -261,6 +261,11 @@ def __init__(self, id_, msg=None):
         self.msg = msg
 
 
+class InvalidAnnexRepositoryError(RuntimeError):
```

Comment:
I wonder if might make sense to get `InvalidRepositoryError` to be a parent class for `InvalidAnnex...` and `InvalidGit...` ?

Reply:
Agree.